[2021-04-15 06:49:52,555] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 06:49:52,555] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:49:52,557] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 06:49:52,562] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 06:49:52,562] - [ Tracker ] - INFO - Target user : thegreat
[2021-04-15 06:49:52,563] - [ Tracker ] - INFO - thegreat is already tracked!
[2021-04-15 06:49:52,563] - [ Tracker ] - INFO - Checking for new followings for thegreat
[2021-04-15 06:49:52,768] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:49:52,994] - [telethon.network.connection.connection] - INFO - The server closed the connection
[2021-04-15 06:49:52,995] - [telethon.network.mtprotosender] - INFO - Connection closed while receiving data
[2021-04-15 06:49:52,995] - [telethon.network.mtprotosender] - INFO - Closing current connection to begin reconnect...
[2021-04-15 06:49:52,995] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:49:53,206] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:49:53,421] - [telethon.network.connection.connection] - INFO - The server closed the connection
[2021-04-15 06:49:53,421] - [telethon.network.mtprotosender] - INFO - Connection closed while receiving data
[2021-04-15 06:49:53,421] - [telethon.network.mtprotosender] - INFO - Closing current connection to begin reconnect...
[2021-04-15 06:49:53,422] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:49:53,625] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:49:54,493] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 06:49:54,495] - [root] - INFO - Bot is ready!
[2021-04-15 06:49:54,498] - [root] - INFO - Queue handler is ready!
[2021-04-15 06:50:02,941] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 06:50:02,942] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:50:02,943] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 06:54:31,967] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 06:54:31,967] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:54:31,970] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 06:54:31,979] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:54:32,177] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:54:33,216] - [root] - INFO - Queue handler is ready!
[2021-04-15 06:54:33,219] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 06:54:33,221] - [root] - INFO - Bot is ready!
[2021-04-15 06:54:38,989] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:54:39,066] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 06:54:39,068] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:54:39,069] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 06:54:49,651] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:54:49,658] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 06:54:49,659] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 06:54:49,668] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:54:49,852] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:54:50,045] - [telethon.network.connection.connection] - INFO - The server closed the connection
[2021-04-15 06:54:50,045] - [telethon.network.mtprotosender] - INFO - Connection closed while receiving data
[2021-04-15 06:54:50,046] - [telethon.network.mtprotosender] - INFO - Closing current connection to begin reconnect...
[2021-04-15 06:54:50,046] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:54:50,245] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:54:51,177] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 06:54:51,178] - [root] - INFO - Bot is ready!
[2021-04-15 06:54:51,243] - [root] - INFO - Queue handler is ready!
[2021-04-15 06:54:56,677] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:03,683] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:10,693] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:17,701] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:24,709] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:31,716] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:38,725] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 06:55:45,735] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 06:55:45,736] - [ Tracker ] - INFO - Target user : tweetyyy
[2021-04-15 06:55:45,736] - [ Tracker ] - INFO - tweetyyy is not tracked yet!
[2021-04-15 06:55:45,737] - [ Tracker ] - INFO - Starting to track tweetyyy
[2021-04-15 06:55:46,538] - [ Tracker ] - INFO - Got 0 total following for tweetyyy
[2021-04-15 06:55:49,549] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 06:55:49,550] - [ Tracker ] - INFO - Target user : tweetyyy
[2021-04-15 06:55:49,550] - [ Tracker ] - INFO - tweetyyy is already tracked!
[2021-04-15 06:55:49,550] - [ Tracker ] - INFO - Checking for new followings for tweetyyy
[2021-04-15 06:55:50,303] - [ Tracker ] - INFO - Got 0 new followings for tweetyyy
[2021-04-15 06:55:53,310] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 06:55:53,311] - [ Tracker ] - INFO - Target user : tweetyyy
[2021-04-15 06:55:53,311] - [ Tracker ] - INFO - tweetyyy is already tracked!
[2021-04-15 06:55:53,311] - [ Tracker ] - INFO - Checking for new followings for tweetyyy
[2021-04-15 06:55:54,135] - [ Tracker ] - INFO - Got 0 new followings for tweetyyy
[2021-04-15 06:55:56,396] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 06:55:56,398] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:55:56,399] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 06:58:57,057] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 06:58:57,069] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 06:58:57,070] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 06:58:57,073] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 06:58:57,073] - [ Tracker ] - INFO - Target user : tweetyyy
[2021-04-15 06:58:57,073] - [ Tracker ] - INFO - tweetyyy is already tracked!
[2021-04-15 06:58:57,073] - [ Tracker ] - INFO - Checking for new followings for tweetyyy
[2021-04-15 06:58:57,073] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 06:58:57,278] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:58:57,794] - [ Tracker ] - INFO - Got 0 new followings for tweetyyy
[2021-04-15 06:58:58,110] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 06:58:58,111] - [root] - INFO - Bot is ready!
[2021-04-15 06:58:58,112] - [root] - INFO - Queue handler is ready!
[2021-04-15 06:59:03,031] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 06:59:03,033] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 06:59:03,035] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 07:02:46,061] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:02:46,073] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 07:02:46,074] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 07:02:46,078] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:02:46,078] - [ Tracker ] - INFO - Target user : tweetyyy
[2021-04-15 07:02:46,078] - [ Tracker ] - INFO - tweetyyy is already tracked!
[2021-04-15 07:02:46,078] - [ Tracker ] - INFO - Checking for new followings for tweetyyy
[2021-04-15 07:02:46,079] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:02:46,376] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:02:47,131] - [ Tracker ] - INFO - Got 0 new followings for tweetyyy
[2021-04-15 07:02:47,516] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 07:02:47,519] - [root] - INFO - Bot is ready!
[2021-04-15 07:02:47,520] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:03:17,165] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:03:27,177] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:03:37,188] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:03:47,201] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:03:57,213] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:04:07,224] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:04:17,233] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:04:27,245] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:04:33,773] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 07:04:33,774] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:04:33,775] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 07:04:35,656] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:04:35,663] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 07:04:35,664] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 07:04:35,669] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:04:35,937] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:04:37,304] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 07:04:37,307] - [root] - INFO - Bot is ready!
[2021-04-15 07:04:37,307] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:04:45,681] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:04:55,693] - [ Tracker ] - INFO - No users in the database!
[2021-04-15 07:05:05,705] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:05:05,705] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:05:05,705] - [ Tracker ] - INFO - rehmanali944228 is not tracked yet!
[2021-04-15 07:05:05,706] - [ Tracker ] - INFO - Starting to track rehmanali944228
[2021-04-15 07:05:07,104] - [ Tracker ] - INFO - Got 25 total following for rehmanali944228
[2021-04-15 07:10:07,212] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:10:07,213] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:10:07,213] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 07:10:07,213] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 07:10:07,214] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:10:08,206] - [ Tracker ] - INFO - Got 0 new followings for rehmanali944228
[2021-04-15 07:15:08,273] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:15:08,274] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:15:08,274] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 07:15:08,274] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 07:15:08,274] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:15:09,346] - [ Tracker ] - INFO - Got 2 new followings for rehmanali944228
[2021-04-15 07:15:09,347] - [ Tracker ] - INFO - Adding new followings to db.
[2021-04-15 07:15:22,227] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 07:15:22,229] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:15:22,229] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 07:15:59,962] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:15:59,969] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 07:15:59,971] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 07:15:59,976] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:15:59,976] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:15:59,976] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 07:15:59,976] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 07:15:59,977] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:16:00,175] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:16:01,086] - [ Tracker ] - INFO - Got 1 new followings for rehmanali944228
[2021-04-15 07:16:01,086] - [ Tracker ] - INFO - Adding new followings to db.
[2021-04-15 07:16:01,089] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:16:01,090] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 07:16:01,091] - [root] - INFO - Bot is ready!
[2021-04-15 07:16:01,774] - [ Tracker ] - INFO - Prepared message for TG : rehmanali944228 just started to follow Asad_Umar.
[2021-04-15 07:16:02,337] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:16:07,956] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 07:16:07,958] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:16:07,958] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 07:28:14,703] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:28:14,711] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 07:28:14,713] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 07:28:14,718] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:28:14,718] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:28:14,718] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 07:28:14,718] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 07:28:14,719] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:28:14,916] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:28:15,811] - [ Tracker ] - INFO - Got 1 new followings for rehmanali944228
[2021-04-15 07:28:15,811] - [ Tracker ] - INFO - Adding new followings to db.
[2021-04-15 07:28:15,814] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:28:15,816] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 07:28:15,817] - [root] - INFO - Bot is ready!
[2021-04-15 07:28:16,527] - [ Tracker ] - INFO - Prepared message for TG : rehmanali944228 just started to follow JahangirKTareen.
[2021-04-15 07:28:17,059] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:28:20,033] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 07:28:20,035] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:28:20,035] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 07:33:10,320] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:33:10,328] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 07:33:10,330] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 07:33:10,334] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:33:10,334] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:33:10,334] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 07:33:10,334] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 07:33:10,334] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:33:10,523] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:33:11,414] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:33:11,420] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 07:33:11,421] - [root] - INFO - Bot is ready!
[2021-04-15 07:33:11,464] - [ Tracker ] - INFO - Got 1 new followings for rehmanali944228
[2021-04-15 07:33:11,464] - [ Tracker ] - INFO - Adding new followings to db.
[2021-04-15 07:33:12,163] - [ Tracker ] - INFO - Prepared message for TG : rehmanali944228 just started to follow [SMQureshiPTI](https://twitter.com/SMQureshiPTI).
[2021-04-15 07:33:12,661] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:33:24,974] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 07:33:24,975] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:33:24,976] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 07:35:57,783] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:35:57,792] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 07:35:57,793] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 07:35:57,799] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 07:35:57,799] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 07:35:57,799] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 07:35:57,799] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 07:35:57,799] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 07:35:58,054] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:35:58,266] - [telethon.network.connection.connection] - INFO - The server closed the connection
[2021-04-15 07:35:58,267] - [telethon.network.mtprotosender] - INFO - Connection closed while receiving data
[2021-04-15 07:35:58,268] - [telethon.network.mtprotosender] - INFO - Closing current connection to begin reconnect...
[2021-04-15 07:35:58,268] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 07:35:58,484] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:35:59,081] - [ Tracker ] - INFO - Got 1 new followings for rehmanali944228
[2021-04-15 07:35:59,082] - [ Tracker ] - INFO - Adding new followings to db.
[2021-04-15 07:35:59,452] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 07:35:59,454] - [root] - INFO - Bot is ready!
[2021-04-15 07:35:59,456] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:35:59,812] - [ Tracker ] - INFO - Prepared message for TG : [rehmanali944228](https://twitter.com/rehmanali944228) just started to follow [MWLorg_Ur](https://twitter.com/MWLorg_Ur).
[2021-04-15 07:36:00,718] - [root] - INFO - Queue handler is ready!
[2021-04-15 07:36:20,770] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 07:36:20,771] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 07:36:20,772] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 15:45:57,104] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 15:45:57,112] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 15:45:57,114] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 15:45:57,139] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 15:45:57,139] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 15:45:57,139] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 15:45:57,139] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 15:45:57,139] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 15:45:57,509] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 15:45:57,900] - [telethon.network.connection.connection] - INFO - The server closed the connection
[2021-04-15 15:45:57,901] - [telethon.network.mtprotosender] - INFO - Connection closed while receiving data
[2021-04-15 15:45:57,901] - [telethon.network.mtprotosender] - INFO - Closing current connection to begin reconnect...
[2021-04-15 15:45:57,901] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 15:45:58,329] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 15:45:59,310] - [ Tracker ] - INFO - Got 0 new followings for rehmanali944228
[2021-04-15 15:46:00,094] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 15:46:00,095] - [root] - INFO - Bot is ready!
[2021-04-15 15:46:00,097] - [root] - INFO - Queue handler is ready!
[2021-04-15 15:46:13,445] - [telethon.client.updates] - ERROR - Unhandled exception on list_user
Traceback (most recent call last):
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/telethon/client/updates.py", line 446, in _dispatch_update
    await callback(event)
  File "/home/virchual/twitterBots/followings_tracker/tg/event_handlers.py", line 51, in list_user
    await event.repond(message)
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/telethon/events/newmessage.py", line 221, in __getattr__
    return getattr(self.message, item)
AttributeError: 'Message' object has no attribute 'repond'
[2021-04-15 15:46:18,258] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 15:46:18,260] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 15:46:18,261] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-15 15:46:26,672] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-15 15:46:26,680] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-15 15:46:26,681] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-15 15:46:26,685] - [ Tracker ] - INFO - Got 1 users from database
[2021-04-15 15:46:26,686] - [ Tracker ] - INFO - Target user : rehmanali944228
[2021-04-15 15:46:26,686] - [ Tracker ] - INFO - rehmanali944228 is already tracked!
[2021-04-15 15:46:26,686] - [ Tracker ] - INFO - Checking for new followings for rehmanali944228
[2021-04-15 15:46:26,686] - [ Tracker ] - INFO - Latest cursor is : -1
[2021-04-15 15:46:27,081] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-15 15:46:28,841] - [ Tracker ] - INFO - Got 0 new followings for rehmanali944228
[2021-04-15 15:46:28,893] - [root] - INFO - Queue handler is ready!
[2021-04-15 15:46:28,899] - [root] - INFO - Setting up event handlers .. 
[2021-04-15 15:46:28,901] - [root] - INFO - Bot is ready!
[2021-04-15 15:46:39,630] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-15 15:46:39,632] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-15 15:46:39,633] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-16 18:13:33,834] - [telethon.network.mtprotosender] - INFO - Connecting to 149.154.167.91:443/TcpFull...
[2021-04-16 18:13:33,839] - [ Tracker ] - INFO - Setting up Apps ...
[2021-04-16 18:13:33,842] - [ Tracker ] - INFO - Followings tracker is ready!
[2021-04-16 18:13:34,151] - [telethon.network.mtprotosender] - INFO - Connection to 149.154.167.91:443/TcpFull complete!
[2021-04-16 18:13:35,303] - [root] - INFO - Setting up event handlers .. 
[2021-04-16 18:13:35,306] - [root] - INFO - Bot is ready!
[2021-04-16 18:13:35,309] - [root] - INFO - Queue handler is ready!
[2021-04-16 18:14:00,377] - [telethon.network.mtprotosender] - INFO - Disconnecting from 149.154.167.91:443/TcpFull...
[2021-04-16 18:14:00,382] - [telethon.network.mtprotosender] - INFO - Disconnection from 149.154.167.91:443/TcpFull complete!
[2021-04-16 18:14:00,383] - [telethon.network.mtprotosender] - INFO - Not disconnecting (already have no connection)
[2021-04-16 18:14:00,570] - [asyncio] - ERROR - Task exception was never retrieved
future: <Task finished name='Task-11' coro=<UpdateMethods._dispatch_update() done, defined at /home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/telethon/client/updates.py:383> exception=KeyboardInterrupt()>
Traceback (most recent call last):
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/telethon/client/updates.py", line 68, in run_until_disconnected
    return self.loop.run_until_complete(self._run_until_disconnected())
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete
    self.run_forever()
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/asyncio/base_events.py", line 1859, in _run_once
    handle._run()
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/telethon/client/updates.py", line 446, in _dispatch_update
    await callback(event)
  File "/home/virchual/twitterBots/followings_tracker/tg/event_handlers.py", line 47, in list_user
    usernames = self.db.get_all_usernames()
  File "/home/virchual/twitterBots/followings_tracker/exts/db.py", line 17, in get_all_usernames
    return [user["username"] for user in users]
  File "/home/virchual/twitterBots/followings_tracker/exts/db.py", line 17, in <listcomp>
    return [user["username"] for user in users]
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/cursor.py", line 1207, in next
    if len(self.__data) or self._refresh():
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/cursor.py", line 1100, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/mongo_client.py", line 1816, in _ensure_session
    return self.__start_session(True, causal_consistency=False)
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/mongo_client.py", line 1766, in __start_session
    server_session = self._get_server_session()
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/mongo_client.py", line 1802, in _get_server_session
    return self._topology.get_server_session()
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/topology.py", line 485, in get_server_session
    self._select_servers_loop(
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/site-packages/pymongo/topology.py", line 226, in _select_servers_loop
    self._condition.wait(common.MIN_HEARTBEAT_INTERVAL)
  File "/home/virchual/anaconda3/envs/telethon/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
